hal/nrf_ppi.h

```diff
@@ -214,6 +214,27 @@ __STATIC_INLINE void nrf_ppi_channel_endpoint_setup(nrf_ppi_channel_t channel,
                                                     uint32_t          eep,
                                                     uint32_t          tep);
 
+/**
+ * @brief Function for setting up event endpoint for a given PPI channel.
+ *
+ * @param[in] eep Event register address.
+ *
+ * @param[in] channel Channel to which the given endpoints are assigned.
+ */
+__STATIC_INLINE void nrf_ppi_channel_set_event_endpoint(nrf_ppi_channel_t channel,
```

Comment:
verb should be placed at the end of the function's name: nrf_ppi_channel_event_endpoint_set

Reply:
same comment to set_task function